internal-registry/che-incubator/command-line-terminal/nightly/meta.yaml

```diff
@@ -12,26 +12,26 @@ firstPublicationDate: "2020-05-13"
 category: Other
 spec:
   endpoints:
-   -  name: command-line-terminal
-      public: true
-      targetPort: 4444
-      attributes:
-        protocol: http
-        type: ide
-        discoverable: false
-        secure: true
-        cookiesAuthEnabled: true
+  - name: command-line-terminal
+    public: true
+    targetPort: 4444
+    attributes:
+      protocol: http
```

Comment:
Since we explicitly run the process with `tls`, does it make sense to specify `https` here?

Reply:
It likely does, but our story on using configuring TLS is kind of unclear -- we also have the `secure` attribute, and as far as I can tell, protocol is converted `http -> https / ws -> wss` based on that property.